test/openst/gnosisSafeModules/delayedRecoveryModule/resetRecoveryOwner.js

```diff
@@ -0,0 +1,118 @@
+// 'use strict';
+
+const assert = require('assert');
+
+const { ContractInteract, Helpers } = require('@openstfoundation/openst.js');
+const shared = require('../../../shared');
+const Deployer = require('../../../../src/deployer');
+const OpenST = require('../../../../src/openst');
+
+describe('DelayedRecoveryModule', async () => {
+  it('resets recovery owner', async () => {
+    const chainConfig = shared.chainConfig;
+    const connection = shared.connection;
+    const web3 = connection.auxiliaryWeb3;
+
+    // Set auxiliaryOrganization address
+    const auxiliaryOrganization = {
+      address: connection.auxiliaryAccount.address,
+    };
+
+    // Add eip20TokenAddress to config for utility token deployment
+    chainConfig.update({
+      eip20TokenAddress: "0x0000000000000000000000000000000000000001",
```

Comment:
Should this be actual OST address from ropsten?

Reply:
I think the answer to this question is arguable, but your question also brought me to the realization I've done more in this test than was strictly necessary to get setup (because I thought more in the OpenST and TokenHolder setups is validated than actually is).

I've removed the extraneous code!

Reply:
We have tested with test/mock data. 
Is it fine to do tests with test argument here in JLP 🤔 ? We should probably check with others also.

Here for example should we be testing with real aux organization and aux utilityBrandedToken contract address deployed on auxiliary chain ❓

Reply:
Do we have any update on above? 
LGTM once above is resolved 👍

Reply:
@abhayks1: apologies, I did not see your comment earlier. There should not be mock data in our smoke tests. As far as this PR, it is an accidental unit test. @pgev will add a comment on this point in response to the conversation in this other thread: https://github.com/OpenST/JLP/pull/60#pullrequestreview-217633011.